src/main/resources/returnyoutubedislike/host/values/strings.xml

```diff
@@ -17,6 +16,10 @@
     <string name="revanced_ryd_dislike_percentage_summary_on">Dislikes shown as percentage</string>
     <string name="revanced_ryd_dislike_percentage_summary_off">Dislikes shown as number</string>
 
+    <string name="revanced_ryd_compact_layout_title">Compact like button layout</string>
+    <string name="revanced_ryd_compact_layout_summary_on">Showing narrow like button</string>
+    <string name="revanced_ryd_compact_layout_summary_off">Showing normal like button</string>
```

Comment:
These do not convey what this switch does properly. Instead, it should say, that it makes it compact by getting rid of the left separator.

Reply:
It does describe what happens: the likes button becomes more narrow.  

The user may not realize what the left separator is.
They'll figure it out when they try turning it on and off.

Reply:
maybe something like:

"Showing minimal like button layout"
"Showing sparse like button layout"

But I feel that 'narrow' describes what happens better.

Maybe some other suggestions?

Reply:
My following suggestion should be formulated/worded better, but something like this maybe:
```suggestion
    <string name="revanced_ryd_compact_layout_title">Compact like button text</string>
    <string name="revanced_ryd_compact_layout_summary_on">Only showing like and dislike separately</string>
    <string name="revanced_ryd_compact_layout_summary_off">Fully formatting like and dislike symmetrically</string>
```

Reply:
maybe:
Compact like button
Like and Dislike displayed together
Like and Dislike displayed separately

Reply:
> Like and Dislike displayed together

But they are not displayed together at all are they?

Reply:
I think this describes it well, and it reads ok too.

![separately](https://user-images.githubusercontent.com/118716522/211218285-54163cea-a334-4661-97e9-249c8b0f0b9c.png)

![together](https://user-images.githubusercontent.com/118716522/211218288-c3231d76-0272-433a-870d-cef22a87c34e.png)

Reply:
Although I'll change it to so "Like**s** and Dislike**s**"

Reply:
> > Like and Dislike displayed together
> 
> But they are not displayed together at all are they?

yeah I guess they are not.  It's "more together" but they still have the • separator between them.

Reply:
Maybe:
Likes and Dislikes minimally styled
Likes and Dislikes fully styled

Reply:
![fully styled](https://user-images.githubusercontent.com/118716522/211218801-cf2c11e5-f500-4236-b098-1553c6391565.png)


![minimally styled](https://user-images.githubusercontent.com/118716522/211218797-a8a1fe9e-557a-4831-9843-c230ff79002a.png)

Reply:
Tough. Maybe 

Likes/ Dislikes buttons styled symmetrically
Likes/ Dislikes button occupy a minimum amount of width

Reply:
"occupy a minimum amount of width" is a bit wordy and might be confusing.

How about:

"Likes and Dislikes styled minimally"
"Likes and Dislikes styled symmetrically"

Reply:
Following on that suggestion, maybe this way:

"Likes and Dislikes styled for minimal width"
"Likes and Dislikes styled for best appearance"

Reply:
yes, I like that best

Reply:
![best appearance](https://user-images.githubusercontent.com/118716522/211219512-bb338405-e93f-48d3-b5aa-7071dd3aeb9c.png)

![minimal width](https://user-images.githubusercontent.com/118716522/211219517-5cbcf01d-3e38-41da-95c8-10b37cfeab89.png)


I think this looks best